tests/integration/pilot/common/apps.go

```diff
@@ -208,12 +208,10 @@ func SetupApps(t resource.Context, i istio.Instance, apps *EchoDeployments) erro
 			WorkloadOnlyPorts: common.WorkloadPorts,
 		}).
 		WithConfig(echo.Config{
-			Service:   PodTproxySvc,
-			Namespace: apps.Namespace,
-			Ports:     common.EchoPorts,
-			Subsets: []echo.SubsetConfig{{
```

Comment:
The annotation shouldn't be removed, right?

Reply:
Oops Left off from test.. thanks for catching this!